omnibus/config/software/more-ruby-cleanup-supermarket.rb

```diff
@@ -26,11 +26,13 @@
 
 build do
   block "Delete bundler git cache, docs, and build info" do
-    vendor_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/")
+    remove_directory "#{install_dir}/embedded/service/supermarket/vendor/cache"
 
-    remove_directory "#{vendor_dir}/bundle/ruby/*/build_info"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/cache"
-    remove_directory "#{vendor_dir}/bundle/ruby/*/doc"
-    remove_directory "#{vendor_dir}/cache"
+    # this expands into the appropriate Ruby release number dir
+    vendor_ruby_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/bundle/ruby/*/")
```

Comment:
remove trailing slash(/) as the paths below includes slash(/)
```suggestion
    vendor_ruby_dir = File.expand_path("#{install_dir}/embedded/service/supermarket/vendor/bundle/ruby/*")
```